clarity_epp/export/sample.py

```diff
@@ -120,3 +120,32 @@ def removed_samples(lims, output_file):
                     stage=removed_stage.name,
                     removed_status=sample_removed_status
                 ))
+
+
+def sample_indications(lims, output_file, artifact_name=None, sequencing_run=None, sequencing_run_project=None):
+    """Export table with sample indications. Lookup samples by sample name or sequencing run (project)."""
+    samples = []
```

Comment:
Is this declaration needed? 
Samples is declared in [if-elif construct (L130-L141)](https://github.com/UMCUGenetics/clarity_epp/blob/8c155092070bb26f27e6958febc8db7e9f90363e/clarity_epp/export/sample.py#L130-L141) afterwards you check [if it is True... (L144)](https://github.com/UMCUGenetics/clarity_epp/blob/8c155092070bb26f27e6958febc8db7e9f90363e/clarity_epp/export/sample.py#L144)

Reply:
Yup otherwise line 144 will crash if nothing is found.